src/renderer-backend-egl.c

```diff
@@ -39,6 +39,9 @@ wpe_renderer_backend_egl_create(int host_fd)
         return 0;
 
     backend->interface = wpe_load_object("_wpe_renderer_backend_egl_interface");
+    if (!backend->interface)
+        return 0;
```

Comment:
I am somewhat surprised that `return NULL;` was not being used in this function before, which IMHO would have been more correct :upside_down_face: 

Probably it's a good moment to start using `NULL` for early returns in this function.

Reply:
@aperezdc Yes I think returning NULL would make sense. I'm not that much familiar with the project and was just following the existing style.

Reply:
I think we can leave this with `return 0;` in this PR (to keep things consistent) and then if we want to change the code to use `NULL` make a follow-up PR with that change.

If @zdobersek does not comment in the next couple of days, I say let's merge this PR as-is :wink: